web-runtime/tests/web/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { FluenceAppService, loadWasm, defaultNames } from '@fluencelabs/marine-js';
```

Comment:
let's add tests with some Marine pure modules? At least I suggest to add tests for the [greeting](https://github.com/fluencelabs/marine/tree/master/examples/greeting) module and port [these](https://github.com/fluencelabs/marine/blob/master/runtime/tests/redis_sqlite.rs) tests for Redis and SQLite.

Reply:
i have committed to help with that